src/components/Resources/index.js

```diff
@@ -5,23 +5,44 @@ import PersonalMenu from '../PersonalMenu';
 import Search from '../Search';
 import Grid from '@material-ui/core/Grid';
 import { ResourceCard } from './ResourceCard';
+import * as helper from '../helper';
 
 function Resources({ getResourcesUrl }) {
   const [resources, setResources] = useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState(null);
 
   useEffect(() => {
     axios
       .get(getResourcesUrl)
       .then(function(response) {
         // handle success
-        setResources(response.data.results);
+        setResources(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
   }, []);
 
+  const search = searchValue => {
+    setSearchValue(searchValue);
+    setLoading(true);
+    setErrorMessage(null);
+    axios
+      .get(helper.buildQueryString(getResourcesUrl, searchValue))
+      .then(function(response) {
+        console.log(response.data);
+        setResources(response.data);
+        setLoading(false);
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  };
+
   return (
     <>
```

Comment:
From what I can tell, this doesn't need to be a Fragment.

Reply:
Hmm...  I didn't write this code, but I did ask for clarification about it.  I'm curious about what's the right answer here... is it really enough to just return <Grid>?

Reply:
@watchtheblur Yes, if <Grid> is the only parent/top-level component!

Reply:
aka you can remove the `<> </>` tags

Reply:
Done in 08cebcf!